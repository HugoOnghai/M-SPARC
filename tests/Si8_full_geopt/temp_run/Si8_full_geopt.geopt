:RELAXSTEP: 1
:R(Bohr):
  9.7714285714E-02   1.9542857143E-01   9.7714285714E-02
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
 -1.2265158673E-02  -1.5034714197E-02  -1.2265853999E-02
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -2.9429604553E+01  -1.9556665034E+00  -2.9042152679E+00 
 -1.9556665034E+00  -2.9654662345E+01  -1.9556799351E+00 
 -2.9042152679E+00  -1.9556799351E+00  -2.9429602788E+01 
:RELAXSTEP: 2
:R(Bohr):
  9.5042709496E-02   1.9215373529E-01   9.5042558041E-02
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
 -1.1893268338E-02  -1.4878073732E-02  -1.1892642441E-02
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -2.9462198051E+01  -1.8806637704E+00  -2.8263379446E+00 
 -1.8806637704E+00  -2.9665220259E+01  -1.8806797242E+00 
 -2.8263379446E+00  -1.8806797242E+00  -2.9462196805E+01 
:RELAXSTEP: 3
:R(Bohr):
  3.5662466384E-02   1.1787107302E-01   3.5665439883E-02
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
 -4.1732683765E-03  -1.0332192003E-02  -4.1729486427E-03
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -2.9960781742E+01  -5.4244006278E-01  -1.3944233312E+00 
 -5.4244006278E-01  -2.9917953215E+01  -5.4241665591E-01 
 -1.3944233312E+00  -5.4241665591E-01  -2.9960780606E+01 
:RELAXSTEP: 4
:R(Bohr):
  9.7583610714E+00   5.5184155579E-03   9.7583676159E+00
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
  1.2127370292E-03  -5.5058820895E-04   1.2121163844E-03
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -3.0106811754E+01   1.4214905637E-01  -6.3998916579E-02 
  1.4214905637E-01  -3.0107738133E+01   1.4221916648E-01 
 -6.3998916579E-02   1.4221916648E-01  -3.0106812778E+01 
:RELAXSTEP: 5
:R(Bohr):
  9.7646170519E+00   4.3869527905E-03   9.7646203583E+00
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
  5.9962695716E-04  -3.9395571259E-04   5.9901838183E-04
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -3.0109285101E+01   7.3869918234E-02  -4.8342193992E-02 
  7.3869918234E-02  -3.0109457757E+01   7.3906191670E-02 
 -4.8342193992E-02   7.3906191670E-02  -3.0109285173E+01 
:RELAXSTEP: 6
:R(Bohr):
  9.7706496737E+00   8.8059586423E-04   9.7706465363E+00
  2.4428571429E+00   2.4428571429E+00   2.4428571429E+00
  0.0000000000E+00   4.8857142857E+00   4.8857142857E+00
  2.4428571429E+00   7.3285714286E+00   7.3285714286E+00
  4.8857142857E+00   0.0000000000E+00   4.8857142857E+00
  7.3285714286E+00   2.4428571429E+00   7.3285714286E+00
  4.8857142857E+00   4.8857142857E+00   0.0000000000E+00
  7.3285714286E+00   7.3285714286E+00   2.4428571429E+00
:F(Ha/Bohr):
  7.1393414808E-05  -8.1238936340E-05   7.1491745544E-05
 -0.0000000000E+00  -0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   9.7714285714E+00   9.7714285714E+00   9.7714285714E+00
:VOLUME:   9.3298397668E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -3.0110368423E+01   8.2308510315E-03  -9.2544688015E-03 
  8.2308510315E-03  -3.0110367513E+01   8.1978852920E-03 
 -9.2544688015E-03   8.1978852920E-03  -3.0110368372E+01 
:RELAXSTEP: 7
:R(Bohr):
  1.0772141265E+01   9.7085694031E-04   1.0772137806E+01
  2.6932500000E+00   2.6932500000E+00   2.6932500000E+00
  0.0000000000E+00   5.3865000000E+00   5.3865000000E+00
  2.6932500000E+00   8.0797500000E+00   8.0797500000E+00
  5.3865000000E+00   0.0000000000E+00   5.3865000000E+00
  8.0797500000E+00   2.6932500000E+00   8.0797500000E+00
  5.3865000000E+00   5.3865000000E+00   0.0000000000E+00
  8.0797500000E+00   8.0797500000E+00   2.6932500000E+00
:F(Ha/Bohr):
  7.5115876960E-05  -8.4895854584E-05   7.5414564855E-05
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0773000000E+01   1.0773000000E+01   1.0773000000E+01
:VOLUME:   1.2502877599E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  4.6740920709E+00   2.0371484373E-03  -2.2897461044E-03 
  2.0371484373E-03   4.6740925580E+00   2.0285015723E-03 
 -2.2897461044E-03   2.0285015723E-03   4.6740920953E+00 
:RELAXSTEP: 8
:R(Bohr):
  1.0648272565E+01   9.5969307004E-04   1.0648269146E+01
  2.6622803562E+00   2.6622803562E+00   2.6622803562E+00
  0.0000000000E+00   5.3245607124E+00   5.3245607124E+00
  2.6622803562E+00   7.9868410686E+00   7.9868410686E+00
  5.3245607124E+00   0.0000000000E+00   5.3245607124E+00
  7.9868410686E+00   2.6622803562E+00   7.9868410686E+00
  5.3245607124E+00   5.3245607124E+00   0.0000000000E+00
  7.9868410686E+00   7.9868410686E+00   2.6622803562E+00
:F(Ha/Bohr):
  7.8411361220E-05  -8.8626867552E-05   7.8725192837E-05
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0649121425E+01   1.0649121425E+01   1.0649121425E+01
:VOLUME:   1.2076506991E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  2.2638542350E+00   2.4964419571E-03  -2.8066961915E-03 
  2.4964419571E-03   2.2638552870E+00   2.4864699324E-03 
 -2.8066961915E-03   2.4864699324E-03   2.2638542717E+00 
:RELAXSTEP: 9
:R(Bohr):
  1.0537653839E+01   9.4972337554E-04   1.0537650455E+01
  2.6346234702E+00   2.6346234702E+00   2.6346234702E+00
  0.0000000000E+00   5.2692469405E+00   5.2692469405E+00
  2.6346234702E+00   7.9038704107E+00   7.9038704107E+00
  5.2692469405E+00   0.0000000000E+00   5.2692469405E+00
  7.9038704107E+00   2.6346234702E+00   7.9038704107E+00
  5.2692469405E+00   5.2692469405E+00   0.0000000000E+00
  7.9038704107E+00   7.9038704107E+00   2.6346234702E+00
:F(Ha/Bohr):
  8.1125519415E-05  -9.1695840836E-05   8.1452817123E-05
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0538493881E+01   1.0538493881E+01   1.0538493881E+01
:VOLUME:   1.1704035842E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -2.3266070158E-01   2.9567455700E-03  -3.3247031883E-03 
  2.9567455700E-03  -2.3265952924E-01   2.9449316025E-03 
 -3.3247031883E-03   2.9449316025E-03  -2.3266065566E-01 
:RELAXSTEP: 10
:R(Bohr):
  1.0548061160E+01   9.5066135242E-04   1.0548057773E+01
  2.6372255078E+00   2.6372255078E+00   2.6372255078E+00
  0.0000000000E+00   5.2744510156E+00   5.2744510156E+00
  2.6372255078E+00   7.9116765233E+00   7.9116765233E+00
  5.2744510156E+00   0.0000000000E+00   5.2744510156E+00
  7.9116765233E+00   2.6372255078E+00   7.9116765233E+00
  5.2744510156E+00   5.2744510156E+00   0.0000000000E+00
  7.9116765233E+00   7.9116765233E+00   2.6372255078E+00
:F(Ha/Bohr):
  8.0863041868E-05  -9.1397055802E-05   8.1187456663E-05
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0548902031E+01   1.0548902031E+01   1.0548902031E+01
:VOLUME:   1.1738747931E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.7123589383E-02   2.9102281568E-03  -3.2741697635E-03 
  2.9102281568E-03   1.7124746586E-02   2.8985935460E-03 
 -3.2741697635E-03   2.8985935460E-03   1.7123643366E-02 
:RELAXSTEP: 11
:R(Bohr):
  1.0547348340E+01   9.5059710838E-04   1.0547344953E+01
  2.6370472887E+00   2.6370472887E+00   2.6370472887E+00
  0.0000000000E+00   5.2740945773E+00   5.2740945773E+00
  2.6370472887E+00   7.9111418660E+00   7.9111418660E+00
  5.2740945773E+00   0.0000000000E+00   5.2740945773E+00
  7.9111418660E+00   2.6370472887E+00   7.9111418660E+00
  5.2740945773E+00   5.2740945773E+00   0.0000000000E+00
  7.9111418660E+00   7.9111418660E+00   2.6370472887E+00
:F(Ha/Bohr):
  8.0883939936E-05  -9.1425974368E-05   8.1209451529E-05
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
 -0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
  0.0000000000E+00  -0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00  -0.0000000000E+00
:CELL:   1.0548189155E+01   1.0548189155E+01   1.0548189155E+01
:VOLUME:   1.1736368239E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.2310303563E-04   2.9138441009E-03  -3.2781709684E-03 
  2.9138441009E-03   1.2390340746E-04   2.9019298577E-03 
 -3.2781709684E-03   2.9019298577E-03   1.2315838269E-04 
