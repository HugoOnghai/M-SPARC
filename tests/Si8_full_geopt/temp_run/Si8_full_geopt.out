***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 18-Sep-2021  Start time: 00:37:47                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.260000 10.260000 10.260000 
LATVEC:
1.000000000000000 0.000000000000000 0.000000000000000 
0.000000000000000 1.000000000000000 0.000000000000000 
0.000000000000000 0.000000000000000 1.000000000000000 
FD_GRID: 26 26 26
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 20
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8_full_geopt.out
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors:
10.260000000000000 0.000000000000000 0.000000000000000 
0.000000000000000 10.260000000000000 0.000000000000000 
0.000000000000000 0.000000000000000 10.260000000000000 
Volume                  :  1.0800455760E+03 (Bohr^3)
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.394615 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8_full_geopt.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.10 7.10 7.10
Number of atoms of type 1          :  8
Estimated total memory usage       :  62.49 MB
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2.548 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
